Dockerfile

```diff
@@ -15,8 +15,16 @@ RUN addgroup -g 1000 rubygems && \
     apk add linux-headers build-base curl openldap-dev git && \
     rm -rf /var/cache/apk/*
 
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.8/main' >> /etc/apk/repositories && \
+    echo 'http://dl-cdn.alpinelinux.org/alpine/v3.8/community' >> /etc/apk/repositories
+
+RUN apk update && \
+    apk add --no-cache libssl1.0
+
 ADD . /application
 RUN chown -R rubygems:rubygems /application
+ADD . /application/data
```

Comment:
This will add the source code to the data folder , are you sure we need this ? 
if you need an empty folder , just create it in the repo and it will be copied with the above commands